test/linalg3.jl

```diff
@@ -161,7 +161,10 @@ for elty in (Float32, Float64, Complex64, Complex128)
                                               0  -0.000000000000002   3.000000000000000])
 end
 
+OLD_STDERR = STDERR
+redirect_stderr(open("/dev/null", "w"))
```

Comment:
I think you can do [this](https://github.com/JuliaLang/julia/blob/8566c4c5b1ef3c3525fd526888f0db9a6df3ce26/test/abstractarray.jl#L452)